LibreOffice supports Box2D 3.1, but its system-library detection only sets
the library name and header for Box2D 2.x.

--- a/configure.ac
+++ b/configure.ac
@@ -12265,6 +12265,11 @@
             _BOX2D_LIB=Box2D
             AC_DEFINE(BOX2D_HEADER,<Box2D/Box2D.h>)
         fi
+    elif test "$BOX2D_MAJOR" -eq "3"; then
+        AC_CHECK_HEADER([box2d/box2d.h], [],
+            [AC_MSG_ERROR([box2d headers not found.])])
+        _BOX2D_LIB=box2d
+        AC_DEFINE(BOX2D_HEADER,<box2d/box2d.h>)
     fi
     AC_CHECK_LIB([$_BOX2D_LIB], [main], [:],
         [ AC_MSG_ERROR(box2d library not found.) ], [])
